app/config/config.yml

```diff
@@ -2,6 +2,7 @@ imports:
     - { resource: parameters.yml }
     - { resource: security.yml }
     - { resource: services.yml }
+    - { resource: "@NewsModuleBundle/Resources/config/services.yml" }
```

Comment:
Dopisująć coś do services.yml utrudniamy sobie tylko kod, ponieważ Symfony ma mechanizmy, aby w inny sposób załadować. Potem, jakbyśmy potrzebowali to dostosować do kogoś innego (to ma być dla wielu NGO) to będziemy musieli w wielu miejscach instalować, a tak to tylko załadujemy Bundla i będzie gotowe. Ktoś nie będzie chciał modułu rejestru to nie załaduje się i gotowe. 
Chyba przy generowaniu Bundla nie kliknąłeś, że chcesz, aby ten Bundl był do rozpowszechniania to ci się tak wygenerowało.

Zwróć uwagę, że wszystkie bundle np. Doctrine ORM też mają swoje servies i są zarejestrowane w Container, ale nie trzeba ich osobno importować. Wystarczy jeden new w AppKernel.php i gotowe. Wygodne i praktyczne. :-)